src/WorkflowManager/Contracts/Models/PayloadDto.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * Copyright 2022 MONAI Consortium
```

Comment:
2023 or is that going to be a later change?

Reply:
I think I remember seeing a comment made somewhere, or maybe it was on a stand-up, about this and I believe they were being kept as is for now and a wider update will be out in place for all the licences.

@woodheadio is this something you know more about?

Reply:
I'm not sure what the requirement for the copyright year is. I don't know if they all need batch updating every year, or whether they should just be updated when a file is updated... whether there is any issue with files that don't change for years not getting their headers updated. Not sure.

Reply:
I've updated this to 2023.